### ISSUE TITLE
expected date for code release?

### ISSUE THREAD

**Issue by yxt132** [raywzy/Bringing-Old-Films-Back-to-Life#4]
It's been quite a while since the author created this repo. Do you have expected date for code release?

**Comment by lucellent** [comment]
I'm eager to know too

**Comment by HelixNGC7293** [comment]
Waiting for code release, so excited!

**Comment by LinixLinux** [comment]
I agree, this project is extremely impressive and ahead of state of the art at this time, and I am eagerly waiting! :)

**Comment by misheeltoli** [comment]
I am waiting for the code release too. Is there anyone tried to implement the code?

**Comment by makemota** [comment]
It looks like authors abandoned this project. After 6 month of creating this repo it shows no signs of life.

**Comment by Tuba202** [comment]
Yeah, even the link to the paper leads to a 404 page. I was really hoping to add this tool to my workflow!
